README.adoc

```diff
@@ -8,6 +8,18 @@ The Khronos Vulkan^®^ Tutorial is based on the "link:https://vulkan-tutorial.co
 
 This repository hosts the contents of the link:https://docs.vulkan.org/tutorial/latest[Khronos Vulkan Tutorial]. The tutorial is part of the link:https://github.com/KhronosGroup/Vulkan-Site[Vulkan Documentation Project].
 
+== Differences
+
+Compared to the original tutorial, this version of the tutorial is teaching up-to-date concepts:
+
+* Vulkan 1.4 as a baseline
+* Dynamic rendering instead of render passes
+* link:https://shader-slang.org/[Slang] as the primary shading language
+* Modern C++ (20)
```

Comment:
Should also say C++ 20 module here (point being it should build fast even in large projects).